modules/plugins/utility/harpoon/default.nix

```
@@ -0,0 +1,6 @@
+_: {
```

```suggestion
{
```

No need for the empty argument placeholder, since this file will never receive arguments in the future.